microsoft-edge/webview2/concepts/user-data-folder.md

```diff
@@ -416,7 +416,7 @@ On this platform, use the default UDF location.
 
 **Example code:**
 
-For example code, see the WinUI 3 `.cs` file, at [WebView2Samples repo > WebView2_WinUI3_Sample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2_WinUI3_Sample).
+For example code, see the WinUI 3 `.cs` file, at [WebView2Samples repo > WebView2_WinUI3_Sample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI3_GettingStarted).
```

Comment:
Weren't the changes in this file already done in a previous PR that we merged? You might need to update your main branch and merge it in this PR branch to make sure this PR branch is up to date.

Reply:
My error.  Not sure why it was added to this pr, but I removed it.

Reply:
I removed this file.  Must have been added accidentally.